configs/components/ruby-augeas.rb

```diff
@@ -46,6 +46,6 @@
 
   pkg.install do
     "chown root:root #{settings[:ruby_vendordir]}/augeas.rb"
-  end if platform.is_solaris?
+  end if platform.is_solaris? || platform.is_huaweios?
```

Comment:
Yep, everything was cherry-picked from master. We can refactor some stuff after the fact.